src/summary/documentation/documentation-section.component.tsx

```diff
@@ -1,30 +1,16 @@
 import React from "react";
 import { match } from "react-router";
-import { getCurrentPatient } from "@openmrs/esm-api";
-import SummarySectionHeader from "../summary-section-header.component";
 import SummarySectionCards from "../cards/summary-section-cards.component";
 import DimensionsCard from "./dimensions-card.component";
 import VitalsCard from "./vitals-card.component";
 
 export default function DocumentationSection(props: DocumentationSectionProps) {
-  const [currentPatient, setCurrentPatient] = React.useState(null);
-
-  React.useEffect(() => {
-    const subscription = getCurrentPatient().subscribe(patient => {
-      setCurrentPatient(patient);
-    });
-
-    return () => subscription.unsubscribe();
-  });
-
   return (
     <>
-      {currentPatient && (
-        <SummarySectionCards match={props.match}>
-          <VitalsCard match={props.match} patient={currentPatient} />
-          <DimensionsCard match={props.match} currentPatient={currentPatient} />
-        </SummarySectionCards>
-      )}
+      <SummarySectionCards match={props.match}>
```

Comment:
These components can now handle being rendered without the patient being loaded yet